tests/test_core.py

```diff
@@ -552,20 +553,20 @@ def test_absolute_filename():
     assert frame_info.filename == full_filename
 
 
-def test_executing_style_defs():
+@pytest.mark.parametrize("expected",
+    [
+        r".c { color: #(999999|ababab); font-style: italic }",
+        r".err { color: #a61717; background-color: #e3d2d2 }",
+        r".c-ExecutingNode { color: #(999999|ababab); font-style: italic; background-color: #ffff00 }",
+        r".err-ExecutingNode { color: #a61717; background-color: #ffff00 }",
+    ]
+)
+def test_executing_style_defs(expected):
     style = style_with_executing_node("native", "bg:#ffff00")
```

Comment:
Maybe a better solution would be to switch from `native` to a different style that hasn't changed recently.

Reply:
Might be, I don't know Pygments well enough. Feel free to suggest a change.